tests/apm/monitoring-topologyChanged-001.phpt

```diff
@@ -0,0 +1,38 @@
+--TEST--
+MongoDB\Driver\Monitoring\TopologyChangedEvent
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_live(); ?>
+<?php skip_if_not_clean(); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$m = create_test_manager();
+
+class MySubscriber implements MongoDB\Driver\Monitoring\SDAMSubscriber
+{
+    public function topologyChanged( \MongoDB\Driver\Monitoring\TopologyChangedEvent $event )
+    {
+        $oid = $event->getTopologyId();
+
+        $topologyDescription = $event->getNewDescription();
+        echo "type: ", $topologyDescription->getType(), "\n";
+        var_dump($topologyDescription->getServers());
+    }
+}
+
+$query = new MongoDB\Driver\Query( [] );
+$subscriber = new MySubscriber;
+
+MongoDB\Driver\Monitoring\addSubscriber( $subscriber );
+
+$cursor = $m->executeQuery( "demo.test", $query );
+?>
```

Comment:
I suggest ending tests with:

```
===DONE===
<?php exit(0); ?>
```

This is beneficial because it allows us to catch failures that may occur after the script intentionally exits. In the case of this test, it looks like a segfault is encountered when PHP is shutting down. After modify these lines locally, I observe the test emit the following output:

```
...

===DONE===
Segmentation fault (core dumped)

Termsig=11
```